src/main/resources/application.yml

```diff
@@ -102,20 +102,7 @@ management:
 prisontonhs:
   only.prisons: ''
 
-aws:
+sqs:
   provider: aws
   region: eu-west-2
 
-sqs:
-  provider: ${aws.provider}
-  queueName: ${sqs.queue.name}
-  queueAccessKeyId: ${sqs.aws.access.key.id}
-  queueSecretAccessKey: ${sqs.aws.secret.access.key}
-  dlqName: ${sqs.dlq.name}
-  dlqAccessKeyId: ${sqs.aws.dlq.access.key.id}
-  dlqSecretAccessKey: ${sqs.aws.dlq.secret.access.key}
-  region: ${aws.region}
```

Comment:
I put everything together here so there is a single place that the SqsConfigProperties are loaded from hopefully making it easier for people to digest.  It also leaves open the possibility of moving SqsConfigProperties into the queue library.  Maybe we could even create the AmazonSQS beans in the queue library config based on SqsConfigProperties. :man_shrugging: